delivery_carrier_label_batch/README.rst

```diff
@@ -17,7 +17,7 @@ If you don't define your pack it will be considered a picking is a single pack.
 Configure
 =========
 
-If using ZPL2 file format, multiple *.zpl can be merged either "as is" or with
+If using ZPL2 file format, multiple \*.zpl can be merged either "as is" or with
```

Comment:
do not edit README.rst, it will be regenerated automatically after the merge, you need to put your changes in readme folder to proper file instead https://github.com/OCA/maintainer-tools/tree/master/template/module/readme